kin-sdk/kin-sdk-lib/src/androidTest/kotlin/kin/sdk/KinAccountIntegrationTest.kt

```diff
@@ -88,6 +87,148 @@ class KinAccountIntegrationTest {
 
     }
 
+    @Test
+    @LargeTest
+    fun getAggregatedBalance_FundAccounts_GotAggregatedBalance() {
+        val kinAccount1 = kinClient.addAccount()
+        val kinAccount2 = kinClient.addAccount()
+        val masterAccount = kinClient.addAccount()
+
+        fakeKinOnBoard.createAccount(kinAccount1.publicAddress.orEmpty(), "10.14159")
+        fakeKinOnBoard.createAccount(kinAccount2.publicAddress.orEmpty(), "100.00301")
+        fakeKinOnBoard.createAccount(masterAccount.publicAddress.orEmpty(), "20.02500")
+
+        linkAccount(kinAccount1, masterAccount, "some package id 1")
+        linkAccount(kinAccount2, masterAccount, "some package id 2")
+
+        // multiply the fee in 4 because there were 4 operations(2 in each linkAccount call)
+        assertThat(masterAccount.aggregatedBalanceSync.value(), equalTo(BigDecimal("130.16960")
+                .subtract(feeInKin.multiply(BigDecimal(4)))))
+    }
+
+    @Test
+    @LargeTest
+    fun getAggregatedBalance_OneOfTheAccountsWasNotCreated_AccountNotFoundException() {
```

Comment:
what do we test here? `getAggregatedBalance` doesn't seems to be executed, and we have tests for account not found

Reply:
Don't know what happened here.
Anyway I fix it.